server/network.go

```diff
@@ -171,22 +174,55 @@ func newLocalNetwork(opts localNetworkOptions) (*localNetwork, error) {
 	}, nil
 }
 
-func createConfigFileString(nodeConfig string, logLevel string, logDir string, dbDir string, pluginDir string, whitelistedSubnets string) (string, error) {
-	var jsonContent map[string]interface{}
-	if err := json.Unmarshal([]byte(nodeConfig), &jsonContent); err != nil {
-		return "", err
+// get the node configs right
```

Comment:
Can we please make this comment a bit more clear as to what it takes in and returns?